examples/example_anisotropy.py

```diff
@@ -0,0 +1,136 @@
+    'idx' : 0,
+    'e' : 0.,                   # reversal potential
+    'syntype' : 'ExpSyn',       # synapse type
+    'tau' : 5.,                 # synaptic time constant
+    'weight' : .001,            # synaptic weight
+    'record_current' : False,    # record synapse current
+}
+
+# Create synapse and set time of synaptic input
+synapse = LFPy.Synapse(cell, **synapse_parameters)
+synapse.set_spike_times(np.array([5.]))
+
+
+# Create a grid of measurement locations, in (um)
+X, Z = np.mgrid[-100:101:2, -100:200:2]
+Y = np.zeros(X.shape)
+
+
+sigma = 0.3
+sigma_tensor = [0.3, 0.3, 0.45]
+
+# Define electrode parameters
+grid_electrode_parameters = {
+    'sigma' : sigma,      # extracellular conductivity
+    'x' : X.flatten(),  # electrode requires 1d vector of positions
+    'y' : Y.flatten(),
+    'z' : Z.flatten(),
+    'method': 'soma_as_point'
+}
+
+grid_electrode_parameters_tensor = {
+    'sigma' : sigma_tensor,      # extracellular conductivity
+    'x' : X.flatten(),  # electrode requires 1d vector of positions
+    'y' : Y.flatten(),
+    'z' : Z.flatten(),
+    'method': 'soma_as_point'
+}
+
+
+# Run simulation, electrode object argument in cell.simulate
+print("running simulation...")
+cell.simulate(rec_imem=True, rec_isyn=False)
+
+# Create electrode objects
+
+grid_electrode = LFPy.RecExtElectrode(cell, **grid_electrode_parameters)
+grid_electrode.calc_lfp()
+
+grid_electrode_tensor = LFPy.RecExtElectrode(cell, **grid_electrode_parameters_tensor)
+grid_electrode_tensor.calc_lfp()
+
+fig = plt.figure(figsize=[10, 5])
+
+ax = fig.add_subplot(121, aspect='equal', xlabel='x', ylabel='z', title="Sigma: %s S/m" % str(sigma),
+                     ylim=[np.min(grid_electrode.z), np.max(grid_electrode.z)],
+                     xlim=[np.min(grid_electrode.x), np.max(grid_electrode.x)])
+
+max_idx = np.argmax(np.abs(cell.imem[0,:]))
+
+LFP = grid_electrode.LFP[:,max_idx].reshape(X.shape)
+im = ax.contourf(X, Z, LFP, 51, vmin=-np.max(np.abs(LFP)) / 1, vmax=np.max(np.abs(LFP)) / 1,
+           cmap='bwr',
+           zorder=-2)
+cbar = plt.colorbar(im)
+
+#plot morphology
+zips = []
+for x, z in cell.get_idx_polygons():
+    zips.append(list(zip(x, z)))
+polycol = PolyCollection(zips, alpha=0.2,
+                         edgecolors='none',
+                         facecolors='k')
+ax.add_collection(polycol)
+
+
+ax.plot(cell.xmid[cell.synidx],cell.zmid[cell.synidx], 'o', ms=5,
+        markeredgecolor='k',
+        markerfacecolor='r')
+
+ax2 = fig.add_subplot(122, aspect='equal', xlabel='x', ylabel='z', title="Sigma: %s S/m" % str(sigma_tensor),
+                     ylim=[np.min(grid_electrode.z), np.max(grid_electrode.z)],
+                     xlim=[np.min(grid_electrode.x), np.max(grid_electrode.x)])
+
+LFP = grid_electrode_tensor.LFP[:,max_idx].reshape(X.shape)
+im = ax2.contourf(X, Z, LFP, 51, vmin=-np.max(np.abs(LFP)) / 1, vmax=np.max(np.abs(LFP)) / 1,
+           cmap='bwr',
+           zorder=-2)
+cbar = plt.colorbar(im)
```

Comment:
add unit on label: 
`cbar.set_label('LFP (mV)'`

Also units on x and y axis labels would be nice. Otherwise, the example runs fine on my end.